amd_openvx/openvx/ago/ago_haf_cpu_arithmetic.cpp

```diff
@@ -3982,9 +3984,9 @@ int HafCpu_Threshold_U1_U8_Binary
 			pixelmask = 0;
 			for (int i = 0; i < 8; i++, width++)
 			{
+				pixelmask >>= 1;
```

Comment:
probably need to add check width<postfixWidth inside for loop as well

Reply:
I've created pull request with needed change at kiritigowda#1